platform-emf/src/main/java/com/softicar/platform/emf/table/IEmfTable.java

```diff
@@ -367,6 +368,11 @@ default void customizeEmfTableConfiguration(EmfTableConfiguration<R, P, S> confi
 		DevNull.swallow(configuration);
 	}
 
+	default void customizeFormAttributesDiv(EmfFormAttributesDivConfiguration<R> attributesDivConfiguration) {
+
+		DevNull.swallow(attributesDivConfiguration);
+	}
+
```

Comment:
Not sure if extending `EmfTableConfiguration` would have been better.

Reply:
I think so. This way we could get rid of `[I]EmfFormAttributesDivConfiguration` which is oddly similar to `[I]EmfFormAttributesDivFactory` anyway.